cypress/integration/my-orcid/addBio.js

```diff
@@ -0,0 +1,39 @@
+/// <reference types="cypress" />
+
+import userData from '../../fixtures/testing_users.json'
+
+describe('My orcid - users are able to add content to their record', async function () {
+  beforeEach(() => {
+    cy.visit(Cypress.env('signInURL'))
+  })
+
+  it('User adds a biography content', function () {
+    const addBio = 'This is my new Bio.'
+    //sign in
+    cy.signin(userData.cyUser_primaryEmaiVerified)
```

Comment:
maybe we can use the camelcase standard or this fixture variable.

Reply:
done